Source/Nyx_hydro.cpp

```diff
@@ -129,11 +129,13 @@ Nyx::just_the_hydro (Real time,
         strang_first_step(time,dt,S_old_tmp,D_old_tmp);
 
 #ifdef _OPENMP
-#pragma omp parallel reduction(max:courno)
+#pragma omp parallel reduction(max:courno) reduction(+:e_added,ke_added)
 #endif
        {
        FArrayBox flux[BL_SPACEDIM], u_gdnv[BL_SPACEDIM];
        Real cflLoc = -1.e+200;
+       e_added = 0.0;
```

Comment:
My understanding is the initialization should be outside omp parallel because they are shared.

Reply:
I wrote it that way because I did not understand initialization rules for reduction variables. However, Section 2.15.3.6 of the OpenMP 4.5 spec (as well as Table 2.7) says that the default initializer for the `+` reduction is 0, which is what we want. So the code that I wrote is not technically wrong, but rather is redundant. I will remove the initializations from inside the parallel region.